stats/opentelemetry/client_metrics.go

```diff
@@ -174,7 +208,23 @@ func (csh *clientStatsHandler) processRPCEnd(ctx context.Context, mi *metricsInf
 		st = canonicalString(s.Code())
 	}
 
-	clientAttributeOption := metric.WithAttributes(attribute.String("grpc.method", ci.method), attribute.String("grpc.target", ci.target), attribute.String("grpc.status", st))
+	attributes := []attribute.KeyValue{
```

Comment:
Can we maybe import the otel packages very-generic-names as `otel*` like `otelattribute` and `otelmetric`?  That would help with readability for me.

Reply:
Done. Will do this in server side PR once we get to that too.